index.html

```diff
@@ -2595,6 +2595,137 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>External Resource Integrity</h2>
+        <p>
+          When including a link to an external resource in a VC, it is 
+          desirable to know whether the resource that is pointed to is the
+          same at signing time as at verification time.  This applies
+          to cases where there is an external resource that is
+          remotely retrieved as well as to cases where the issuer and/or
+          verifier may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
+          context(s) used in the verifiable credential are the same when
+          used by both the issuer and verifier. 
+        </p>
+        <p>
+          To validate that a resource referenced by a Verifiable Credential is
+          the same at verification time as at issuing time, an implementer
+          MAY include a property named <code>resourceIntegrity</code> that
+          stores an array of objects that describe additional integrity
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the [[URL]] to the resource named <code>id</code> and the 
+          <code>integrity</code> information for the resource
+          constructed using the method specified in <a
+          href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource
+          Integrity</a>.
+        </p>
+        <p>
+          There MUST NOT be more than one object in the
+          <code>resourceIntegrity</code> per <code>id</code>. 
+        </p>
+        <p>
+          When performing integrity checking the
+          strongest hash algorithm available SHOULD be used to confirm
+          integrity of the resource.
```

Comment:
Thanks for that! So the following is possible. There's something that bothers me of including an array in a single string. 
```
{
  "image": {
    "id": "https://www.w3.org/StyleSheets/TR/2016/logos/W3C",
    "integrity": "sha384-+EzHAeRZGsx0HtzBd6u0Qy5ezQPL1yUD75E3TZjI/r9UJmecM25khdGE+VWaYJUW  sha256-5TGRFv0PjQqBlN5hww6affFMs2cqC9wkxvIlbG4tJ0k="
  }
}
```

Reply:
Ahh, right, I forgot this about SRI. Can we just use an array of single values? When SRI was designed, they were limited to a single property, so didn't have the option of putting the values in an array like we do. Any opposition to doing that (since SRI code can still be used to interpret this value)?

Reply:
Also agree with @andresuribe87 that the language needs to be changed to something like: "If multiple `sriDigest` values exist, then the most computationally difficult cryptographic hash to defeat SHOULD be used."

Reply:
happy to revert the changes that were incorporated to account for requests from DB